irohad/torii/impl/command_service_transport_grpc.cpp

```diff
@@ -185,20 +185,14 @@ namespace torii {
     std::string client_id =
         (client_id_format % context->peer() % hash.toString()).str();
 
-    // in each round, increment the round counter, showing number of consecutive
-    // rounds without status update; if it becomes greater than some predefined
-    // value, stop the status streaming
-    std::atomic_int round_counter{0};
-    consensus_gate_->onOutcome().subscribe(
-        [this, &subscription, &round_counter](const auto &) {
-          auto new_val = round_counter.load() + 1;
-          if (new_val >= maximum_rounds_without_update_) {
-            subscription.unsubscribe();
-          } else {
-            round_counter++;
-          }
-        });
+    auto consensus_gate_observable =
+        consensus_gate_objects_
+            // a dummy start_with lets us don't wait for the consensus event
+            // on further combine_latest
+            .start_with(ConsensusGateEvent{});
 
+    boost::optional<iroha::protocol::TxStatus> last_tx_status;
+    auto rounds_counter{0};
```

Comment:
It is used only in `take_while` lambda. Can't it be made static there and not be declared here?